editor/app/components/slider.jsx

```diff
@@ -0,0 +1,62 @@
+import * as React from "react";
+import "./slider.css";
+
+/**
+ *
+ * @param {number} start - The start value of the range.
+ * @param {number} end - The end value of the range.
+ * @param {number} step - The increment given to each step.
```

Comment:
Thank you for these docs!!  Im having a lil hard time understanding `step`.

So if you have a range like 1 - 10, and step is 1, that means we are going to split 1-10 into chunks of 1 unit?  So we end up with `[1,2,3,4,5,6,7,8,9,10]`?

And if step is 2 then you end up with `[1, 3, 5, 7, 9]` or something like that?

Do you anticipate us using `step` for any value other than 1?   If not - maybe we can skip that as an option for now and we can add it when the tie comes?

Im fine landing like this, but curious what you think about adding a couple of unit tests, if nothing to test the two examples I provided above, and maybe one with a negative min.